Documentation/user-guides/cluster-monitoring.md

```diff
@@ -480,6 +480,49 @@ spec:
 
 Read more in the [alerting guide](alerting.md) on how to configure the Alertmanager as it will not spin up unless it has a valid configuration mounted through a `Secret`. Note that the `Secret` has to be in the same namespace as the `Alertmanager` resource as well as have the name `alertmanager-<name-of-alertmanager-object` and the key of the configuration is `alertmanager.yaml`.
 
+## Basic auth
```

Comment:
Ok! I splited the Basic Auth onto a new file and sorry for the `embedmd` I just forgot to add the files to git.